gui/src/installer/mod.rs

```diff
@@ -164,10 +173,36 @@ impl Installer {
 pub async fn install(ctx: Context) -> Result<PathBuf, Error> {
     let mut cfg: liana::config::Config = ctx.extract_daemon_config();
     // Start Daemon to check correctness of installation
-    let daemon = liana::DaemonHandle::start_default(cfg.clone()).map_err(|e| {
-        Error::Unexpected(format!("Failed to start daemon with entered config: {}", e))
-    })?;
-    daemon.shutdown();
+    match liana::DaemonHandle::start_default(cfg.clone()) {
+        Ok(daemon) => daemon.shutdown(),
+        Err(e) => match e {
+            StartupError::Bitcoind(BitcoindError::Wallet(path, error)) => {
+                if !matches!(error, WalletError::Creating(_)) {
```

Comment:
Maybe gui should still try to delete even if the error is a wallet creation error (for example a previous wallet already exists)

Reply:
Yes i think if there is an error regarding the wallet the GUI should always try once to delete the watchonly wallet and retry. Or maybe back it up? To avoid a long rescan.